### DISCUSSION TITLE
Annotations From PDF's files not showing

### DISCUSSION THREAD

**Discussion by Leodqa** [hadynz/obsidian-kindle-plugin#276] [Q&A]
Hello.

For some reason my annotations from my pdf files synced from My clippings, aren't showing along with my highligths. However ebooks synced from amazon shows the annotations from the highligths just fine.

This is some mistake of mine from the coding? There is a way to make them appear? Or the plugin don't show the annotations from PDF's in it's current update?

**Comment by aalejandroaraujo** [comment]
That's exactly the way this plugin works. It's clearly stated in the notes

**Reply by Leodqa** [reply]
I think I didn't explain my problem correctly since english is not my main language, but the plugin description states that notes from any book apears via my clippings method. 

In that part: 

**"for highlights from books, articles, PDFs, and personal documents
not purchased from Amazon (see next method).

Kindle Device (My Clippings)

Sync your highlights by uploading your My Clippings.txt file stored on your Kindle device.
This file includes highlights, bookmarks and notes for any book on your Kindle regardless
if it has been purchased via Amazon.

You can extract your My Clippings.txt file by plugging it into your computer using USB."**

In my case I just see the highlights from PDFs, I don't see the notes I take within these highlights. And sense the description of the plugins states that it should show from any book. It's possibly that maybe I did something wrong, making the plugin not optimal for me. Or just that PDF specifically really don't work.
